Tests/Testo/Invoice/PaymentInformation/AmazonPayPaymentServiceTest.php

```
@@ -0,0 +1,313 @@
+{
+    private function makeTempPemDir(bool $withPrivateKey): string
+    {
+        $dir = sys_get_temp_dir() . '/amazon_pay_test_' . uniqid();
+        mkdir($dir, 0777, true);
+        if ($withPrivateKey) {
+            $resource = openssl_pkey_new([
+                'private_key_bits' => 2048,
+                'private_key_type' => OPENSSL_KEYTYPE_RSA,
+            ]);
+            if ($resource === false) {
+                throw new \RuntimeException('Unable to generate a test RSA key.');
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '40', column '27'). (undefined)

(MissingImport)